### PR TITLE
WIP: 2021 03 29 adrian01c htmt css clocking

### PR DESCRIPTION
**Pull request by Adrian01C**

### CODE REVIEW

**Comment by adelacecilia** [comment]
Consider re-doing the indentation :)

**Review by adelacecilia — COMMENTED** [review]
Consider re-doing the indentation! :)

File: web/adrian01c/classes/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.city {
+  background-color:grey;
+  color: beige;
+  border: 15px solid plum;
+  margin: 25px:verdana;
+}
+</style>
+</head>
+<body>
+
```
Comment (vasilemaftita) [inline web/adrian01c/classes/index.html]: please delete the empty line... no empty lines in HTML.

File: web/adrian01c/classes/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<div class="city">
+<h2>Sibiu</h2>
+<p>Sibiu is the capital of culture in Romania.</p>
+</div> 
+
```
Comment (vasilemaftita) [inline web/adrian01c/classes/index.html]: please delete the empty line... no empty lines in HTML.

File: web/adrian01c/classes/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<div class="city">
+<h2>Bucharest</h2>
+<p>Bucharest is the capital of Romania.</p>
+</div>
+
```
Comment (vasilemaftita) [inline web/adrian01c/classes/index.html]: please delete the empty line... no empty lines in HTML.

File: web/adrian01c/classes/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (vasilemaftita) [inline web/adrian01c/classes/index.html]: Please indent ` <head>` properly

File: web/adrian01c/our-promise.html/about.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+						<div><a href="https://github.com/marianzburlea/tesla-saves-lives">Github project</a></div>
+						<div><a href="https://github.com/marianzburlea/tesla-saves-lives/blob/master/SOURCES.md">Data sources</a></div>
+						<da-mi-cartea-ta-de-credit style="position: relative; width:100%; padding-top: 56.25%">
+								<iframe width="560" height="315" src="https://www.youtube.com/embed/Y6Gz1ceMDNE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+						</da-mi-cartea-ta-de-credit>
+				</td>
+			</tr>
+		</table>
+	</body>
+</html>
```
Comment (vasilemaftita) [inline web/adrian01c/our-promise.html/about.html]: you need a new line added at the end of HTML

**Review by vasilemaftita — DISMISSED** [review]
you need a new line added at the end of HTML

**Review by RadosavAntonio — DISMISSED** [review]
I recommend a smaller indentation.
https://stackoverflow.com/questions/34174207/how-to-change-indentation-in-visual-studio-code 
Also please leave an empty line at the end of each file.

**Review by QuintyHH — APPROVED** [review]
Nice

**Review by RalucaRbc — APPROVED** [review]

**Review by screpyy96 — APPROVED** [review]
💯

**Pull request merged**